app/lint-baseline.xml

```diff
@@ -36,7 +25,7 @@
 
     <issue
         id="RestrictedApi"
-        message="ListenableFutureKt.await can only be called from within the same library group (referenced groupId=`androidx.work` from groupId=`wire-android`)"
+        message="ListenableFutureKt.await can only be called from within the same library group (referenced groupId=`androidx.work` from groupId=`wire-android-reloaded`)"
```

Comment:
We are no longer reloaded 😁 The old project is archived

Reply:
For this case I think we cannot do much, since this is an autogen file, it should be fine either. Might be related to the IDE settings (maybe @Garzas you call it reloaded locally or cloned it as is).

But that's true, we keep unconsciously naming it reloaded 😅

Reply:
Oops 😅